docs/time/index.rst

```diff
@@ -1270,6 +1270,29 @@ for example, may be more convenient to use to create `~datetime.tzinfo`
 objects used to specify timezones rather than the `~astropy.time.TimezoneInfo`
 object.
 
+strftime and strptime
+---------------------
+Sometimes you want the time represented by a `~astropy.time.Time` object to be custom
```

Comment:
Replace this text with:
```
The `~astropy.time.Time` object supports output string representation
using the `format specification language
<https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior>`_
defined in the Python standard library for `datetime.strftime`.
This can be done using the `~astropy.time.Time.strftime`
method::
```